### DISCUSSION TITLE
Auto Exposure Bracketing for Still Photos in GoPro 10, 10 Bones or 11.

### DISCUSSION THREAD

**Discussion by FotoHobby1** [gopro/labs#788] [Feature Requests]
I'm seeking an auto exposure (bracketed exposure) functionality as a soon-to-be owner of either GoPro 10, GoPro 10 Bones, or GoPro 11. I'm also thinking this feature would be desired by other exisiting GoPro Labs users.

For my primary use case, an auto exposure mode would be configured to trigger the GoPro to take 5 still exposures each at 2EV increments in this sequence -4 EV >-2EV >0EV >+2EV >+4EV . I would be using this setting while the GoPro camera is attached to an FPV drone, so I can have my drone also be hybrid video / phtography aerial platfrom. 

To have this as Auto Exposure Bracketing setting that would be able to be reused when needed, and  always on in the current active session in Photo / Still mode, and to have it output the images not as already merged, but as separate images to be processed manually outside of the GoPro system is the functionality I am looking for. It would be assumed in the details I lack, that such images would output to the highest quality raw format possible for processing.

P.S. I have searched the discussions for bracketing requests and coulld not find...forgive me if already discussed.

**Comment by dnewman-gpsw** [comment]
GoPro's don't have bracketing, but you can fake it in Labs, yet there about 2 seconds between images. Likely not ideal for drones.

**Reply by FotoHobby1** [reply]
Hi Thanks for your response. 2 seconds between brackets seems pretty normal. If I place my drone in a position hold, on most calm days the movement between shots can be handled externally by the HDR merge software.

**Reply by dnewman-gpsw** [reply]
To be clear, 5 shoots with EV -4 thru +4, took a total of 10 seconds on a HERO11.

Using this QR Command
```x0!Sx-2!Sx-4!Sx2!Sx4!Sx0```

**Reply by FotoHobby1** [reply]
Thank you !
